pkgs/development/python-modules/inflate64/default.nix

```
@@ -0,0 +1,35 @@
+{
+  buildPythonPackage,
+  fetchFromGitea,
+  lib,
+  setuptools,
+  setuptools-scm,
+}:
+
+buildPythonPackage rec {
+  pname = "inflate64";
+  version = "1.0.1";
+
+  pyproject = true;
+
+  src = fetchFromGitea {
+    domain = "codeberg.org";
+    owner = "miurahr";
+    repo = pname;
```

`repo = pname;` is an antipattern as these two fields are arbitrary and not necessarily related
```suggestion
    repo = "inflate64";
```